bundles/net.tourbook/src/de/byteholder/geoclipse/map/LocalResponseCache.java

```diff
@@ -68,9 +68,6 @@ private static boolean isUpdateAvailable(final URI remoteUri, final File localFi
 		URLConnection conn;
 		try {
 			conn = remoteUri.toURL().openConnection();
-		} catch (final MalformedURLException ex) {
-			ex.printStackTrace();
-			return false;

```

Comment:
Because it's already caught in the IOException.
See Eclipse message 
![image](https://user-images.githubusercontent.com/25235665/81329571-0300da80-905c-11ea-9a79-814d4b856c81.png)